app/experimenter/experiments/models.py

```diff
@@ -76,6 +76,13 @@ class Experiment(ExperimentConstants, models.Model):
         validators=[MaxValueValidator(ExperimentConstants.MAX_DURATION)],
     )
 
+    addon_name = models.CharField(max_length=255, blank=True, null=True)
+    addon_experiment_id = models.CharField(
+        max_length=255, blank=True, null=True
+    )
+    addon_testing_url = models.URLField(blank=True, null=True)
```

Comment:
Are you confident this default length is enough? It is 200 chars as of today in Django 2. 
I have, so many times, fallen for that default only to realize later that it wasn't enough. 
Especially when the URLs are quite unpredictable and especially if the URLs *might* contain Unicode which when turned into quoted hex becomes super long. 

I'm not super familiar with what these URLs are but something like `max_length=400` would give me a lot better sleep at night.